views/js/qtiCreator/widgets/helpers/formElement.js

```diff
@@ -282,7 +282,10 @@ define([
                     updateResponseDeclaration(element, value, options.updateCardinality);
                 }
 
-                if (!value && (element.is('orderInteraction') || element.is('graphicOrderInteraction'))) {
+                if (name === 'upperBound' && this.disabled) {
+                    // if the field is disabled, the corresponding attribute should be removed.
+                    element[options.attrMethodNames.remove](name);
+                } else if (!value && (element.is('orderInteraction') || element.is('graphicOrderInteraction'))) {
```

Comment:
Maybe we can use something like this?
```suggestion
                if (!value && (element.is('orderInteraction') || element.is('graphicOrderInteraction') || element.is('responseDeclaration'))) {
```

Reply:
Don't think so, but not sure. Because we should remove value only for the disabled `upperBound` field. While the suggested solution may affect different field and interactions

Reply:
I'm trying to understand if `upperBound` is also used in another interactions like hotspot 🤔